.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+| Q | A
+| --- | ---
+| Bug? | no
+| Critical? | no
+| Version Used | 1.1.x
```

Comment:
The exact commit sha would definitely  be more useful. So I would say here should be a specific tag or a commit sha.